src/services/deviationsFeeds/PriceDataOverflowChecker.ts

```diff
@@ -0,0 +1,42 @@
+import {inject, injectable} from 'inversify';
+
+import {PriceData} from '../../types/DeviationFeeds';
+import {Logger} from 'winston';
+
+@injectable()
+export class PriceDataOverflowChecker {
+  @inject('Logger') logger!: Logger;
+
+  apply(priceData: PriceData): boolean {
+    const {data, price, heartbeat, timestamp} = priceData;
+
+    let valid = true;
+
+    const dataMaxUint8 = 2 ** 8;
+    const heartbeatMaxUint24 = 2 ** 24;
+    const timestampMaxUint32 = 2 ** 32;
+    const priceMaxUint128 = 2n ** 128n;
+
+    if (data >= dataMaxUint8) {
+      valid = false;
+      this.logger.error(`[PriceDataOverflowChecker] data: got ${data}, max uint8 ${dataMaxUint8}`);
+    }
+
+    if (heartbeat >= heartbeatMaxUint24) {
+      valid = false;
+      this.logger.error(`[PriceDataOverflowChecker] heartbeat: got ${heartbeat}, max uint8 ${heartbeatMaxUint24}`);
```

Comment:
Typo on the following three logs. Shouldn't be uint24, uint32 and uint128?

Reply:
done